cmd/ocagent/cli.go

```diff
@@ -15,17 +15,22 @@
 package main
```

Comment:
The existence of this file is making `go run` awkward for ocagent, one needs to have both main.go and cli.go on the command... I think I will merge them...

Reply:
Do you have a sense on how much more work it would be to just unify these files with the occollector startup files?

Reply:
It will be better to unify config first, then it becomes easier.

Reply:
deleted cli.go